lib/api/web-element/commands/getRect.js

```diff
@@ -27,11 +27,11 @@
  * @memberof ScopedWebElement
  * @instance
  * @syntax browser.element(selector).getRect()
- * @see https://www.w3.org/TR/webdriver#dfn-get-element-rect
+ * @link /#dfn-get-element-rect
  * @returns {ScopedValue<{ width: number, height: number }>}
  * @alias getSize
  * @alias getLocation
  */
-module.exports.command = function() {
+module.exports.command = function () {
```

Comment:
I suggest, remove this unnecessary space between as it shows no significant change.